tests/wpt/mozilla/tests/css/flex-auto-min-size-fallback.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html><head><title>non-flexbox | computed style | min-width: auto</title>
+<link href="http://www.w3.org/TR/css-flexbox-1/#min-size-auto" rel="help">
```

Comment:
Please don't use `TR` links for CSS specs, use https://drafts.csswg.org/css-flexbox/.